ceci/minirunner.py

```diff
@@ -175,7 +185,7 @@ def __init__(self, nodes, job_graph, log_dir):
         self.queued_jobs = list(job_graph.keys())
 
 
-    def run(self, interval):
+    def run(self, interval, timeout=1e300):
```

Comment:
That number jumped out to me, but I guess it's ok, the default behavior is just no timeout